### PR TITLE
Support config changing and service stop [ECR-3781]

### PR DESCRIPTION
**Pull request by popzxc**
It is now possible to change service config or stop service instance in `exonum-launcher` (only in `simple` mode yet).

`InstanceSpecLoader` obtained a new method `serialize_config` for serializing configs for config change request.

Now launcher config can look as the following:

```yaml
#...

instances:
  xnm-token:
    artifact: cryptocurrency
    action: start # <- start instance (default, works the same as before)
  nnm-token:
    artifact: cryptocurrency
    action: stop # <- stop service instance (service ID will be deduced by launcher)
  configuration:
    artifact: configuration
    action: config # <- change config
    config:
       mode: 1
```

### CODE REVIEW

**Comment by vitvakatu** [comment]
It seems to me that the format of configuration file isn't obvious.
What about
```yaml
start:
  artifact1:
    instance1
  artifact2:
    instance2
stop:
  artifact1:
    instance3
configure:
   artifact3:
     instance4
       config:
```

It can be done separately, of course, just thinking.

**Comment by popzxc** [comment]
I thought about it, but the current state allows user to adapt existing config easily: e.g., you have a config to deploy & start some service, and you want to change the configuration of this service.

With current structure, you only have to add "deploy: false" to artifact, change the action to "config" and change configuration params, the same for scenario when you stop service, and then enable it again.

If we will create separate sections, it won't be as handy, I think.

File: exonum_launcher/configuration.py [hunk 1]
```diff
@@ -98,9 +108,39 @@ def __init__(self, data: Dict[Any, Any]) -> None:
...
+        for field in expected_fields:
+            if field not in self.consensus:
+                raise RuntimeError(
+                    f"Invalid consensus config '{self.consensus}', at least the field '{field}' is missing."
+                )
+
+        for key_pair in self.consensus["validator_keys"]:
+            if len(key_pair) != 2:
+                raise RuntimeError(
+                    "Validaor keys should be a list of pairs (consensus_key, service_key) in hexadecimal form"
```
Comment (Kormick) [inline exonum_launcher/configuration.py]:
```suggestion
                    "Validator keys should be a list of pairs (consensus_key, service_key) in hexadecimal form"
```

**Comment by vitvakatu** [comment]
Sounds reasonable, yep. Perhaps we can think about it in the future.

**Review by Kormick — APPROVED** [review]

**Review by aleksuss — APPROVED** [review]
👍🏻

**Pull request merged**